src/actinia_core/core/common/config.py

```diff
@@ -774,9 +792,27 @@ def read(self, path=DEFAULT_CONFIG_PATH):
                 if config.has_option("MISC", "LOG_LEVEL"):
                     self.LOG_LEVEL = config.getint("MISC", "LOG_LEVEL")
                 if config.has_option("MISC", "SAVE_INTERIM_RESULTS"):
-                    self.SAVE_INTERIM_RESULTS = config.getboolean(
-                        "MISC", "SAVE_INTERIM_RESULTS"
-                    )
+                    save_interim = config.get("MISC", "SAVE_INTERIM_RESULTS")
+                    if save_interim == "True":
+                        self.SAVE_INTERIM_RESULTS = True
+                    elif save_interim == "False":
+                        self.SAVE_INTERIM_RESULTS = False
+                    else:
+                        self.SAVE_INTERIM_RESULTS = save_interim
```

Comment:
I find it a bit confusing that a config flag has different options which are differen data types. Would it be better to have maybe `onError`, `always` and `never` ? This way, all options are strings. But then it is not backward compatible anymore, so open for discussions here.

Reply:
As discussed, leave as is.